manim/__main__.py

```diff
@@ -62,6 +62,14 @@ def main():
             else:
                 logger.error("No subcommand provided; Exiting...")
 
+        elif args.cmd == "plugins":
+            from manim.plugins import plugins_flags
+
+            if args.list:
+                plugins_flags.list_plugins()
+            elif not args.list or args.update:
```

Comment:
In the current state of this PR, `update()` is never being used, so the check for `args.update` is kind of unnecessary, no? If so, please remove it and we can add it again later when necessary :slightly_smiling_face:

Reply:
Ahh yes, I forgot the implementation too. Will remove